Buddies/MyActivities/MyActivitiesVC.swift

```diff
@@ -7,20 +7,93 @@
 //
 
 import UIKit
+import FirebaseAuth
+import Firebase
+class MyActivitiesVC: ActivityTableVC {
 
-class MyActivitiesVC: UITableViewController {
-    var fab: FAB!
+    var user: User?
+    var cancelUserListener: Canceler?
+    
+    func loadCurUserActivities(userID: String){
+        
+        var created = [ActivityId]()
+        var joined = [ActivityId]()
+        var previous = [ActivityId]()
+        
+        //Get activities current user is associated with
+        FirestoreManager.shared.db.collection("activities").whereField("members", arrayContains: userID).getDocuments(){(querySnapshot, err) in
```

Comment:
Perhaps this could be abstracted to a static member of FirestoreManager? Maybe it would look like:

```swift
FirestoreManager.getOwnActivities() { created, joined previous in
  self.displayIds = [created, joined, previous]
  self.loadData(for: self.displayIds)
}
```

Reply:
Done in latest commit.